system-tests/ft-suite/src/test/kotlin/com/hotels/styx/routing/LoadBalancingGroupSpec.kt

```diff
@@ -149,30 +149,30 @@ class LoadBalancingGroupSpec : FeatureSpec() {
                                   app-A-01:
                                     type: HostProxy
                                     tags:
-                                      - App-A
+                                      - lbGroup=App-A
                                       - state:inactive
                                     config:
                                       host: localhost:${appA01.port()}
 
                                   app-A-02:
                                     type: HostProxy
                                     tags:
-                                      - App-A
+                                      - lbGroup=App-A
                                       - state:inactive:3
```

Comment:
shouldn't this use `:` instead of `=` to be consistent with the state:inactive tag?

Reply:
Or are we going to standardize on `=`?

Reply:
Hi. I thought about this and chose ’=‘ because ‘:’ already appears in creation time tag values.

Therefore would be best to standardise on ‘=‘. My gut feeling is that it will be less likely to occur in the value parts.